examples/plot_hawkes_conditional_law.py

```diff
@@ -32,10 +32,12 @@
     kernels=[[
         HawkesKernelPowerLaw(multiplier[0], cutoff, exponent, support),
         HawkesKernelPowerLaw(multiplier[1], cutoff, exponent, support)
-    ], [
-        HawkesKernelPowerLaw(multiplier[2], cutoff, exponent, support),
-        HawkesKernelPowerLaw(multiplier[3], cutoff, exponent, support)
-    ]], baseline=[0.05, 0.05], seed=382, verbose=False)
+    ],
```

Comment:
Why so many files have been badly linted ? (changed indents and such ?

Reply:
This is correct formatting based on your yapf config